elementary/monitor/dbt_project/macros/alerts/get_source_freshness_alerts.sql

```diff
@@ -1,5 +1,11 @@
 {% macro get_pending_source_freshness_alerts(days_back) %}
     -- depends_on: {{ ref('alerts_source_freshness') }}
+    {% set alerts_source_freshness_relation = ref('alerts_source_freshness') %}
+    {% set error_after_column_exists = elementary.column_exists_in_relation(alerts_source_freshness_relation, 'error_after') %}
```

Comment:
Can you get the columns once with `adapter.get_columns_in_relation` and then check for the required columns?
We're re-running it each time in `column_exists_in_relation`.

Here's how I'd do it.
At `column_exists_in_relation`, I'd add `columns=none`, and they load them if they're `none`, otherwise use `columns`.